stellar-dotnet-sdk/SorobanTransactionMeta.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Linq;
+
+namespace stellar_dotnet_sdk;
+
+/// <summary>
+///     Holds the Soroban transaction metadata.
+/// </summary>
+public class SorobanTransactionMeta
+{
+    /// <summary>
+    ///     We can use this to add more fields.
+    /// </summary>
+    public ExtensionPoint ExtensionPoint { get; private init; } = new ExtensionPointZero();
+
+
+    public SCVal ReturnValue { get; private init; } = new SCVoid();
+
+    /// <summary>
+    ///     Custom events populated by the contracts themselves. One list per operation.
+    /// </summary>
+    public ContractEvent[] Events { get; private init; } = Array.Empty<ContractEvent>(); // TODO Unit test
+
+    /// <summary>
+    ///     Diagnostics events that are not hashed. One list per operation.
+    ///     This will contain all contract and diagnostic events. Even ones that were emitted in a failed contract call.
+    /// </summary>
+    public DiagnosticEvent[] DiagnosticEvents { get; private init; } = Array.Empty<DiagnosticEvent>();
+
+    /// <summary>
+    ///     Creates the corresponding <c>SorobanTransactionMeta</c> object from an <c>xdr.SorobanTransactionMeta</c> object.
+    /// </summary>
+    /// <param name="xdrSorobanTransactionMeta">An <c>xdr.SorobanTransactionMeta</c> object to be converted.</param>
+    /// <returns>A <c>SorobanTransactionMeta</c> object. Returns null if the provided object is null.</returns>
```

Comment:
It's just a "code" markup, just like in Slack or other messaging apps. I think it's better than just plain text, and it's more logical to use <c> instead of <see> when we are referring to something that is already "linkable" in the method/class signature. It'll be easier for me to explain this in a sharing screen session.